fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -2717,6 +2722,346 @@ Future<Optional<std::string>> DataDistributionImpl::commit(ReadYourWritesTransac
 	return msg;
 }
 
+bool existingBulkLoadUpdate(ReadYourWritesTransaction* ryw, KeyRange range) {
+	KeyRange rangeToCheck =
```

Comment:
This returns true if a bulk loading task, status or cancel. Is that what you want?